_posts/2020-01-20-pulp-3-GA-demos.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Pulp 3.0 GA Demos
+author: Dana Walker
+tags:
+  - 3.0
+---
+To go along with Pulp 3.0 GA, we have prepared a playlist of demos to cover features in pulpcore on
+the [Pulp YouTube Channel](https://www.youtube.com/pulpproject).
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PLwm8_O6oKSS0C3zlo8gFkPqXpMkAI7rCD" frameborder="0" allowfullscreen></iframe>
+
+Check it out and let us know if you find it useful.  Also, if there is any content you don't see but
+would like a demo to cover, please reach out to us with that request.
```

Comment:
Can we identify how they will reach out? I think via pulp-list@redhat.com is probably what I would recommend. Maybe link to https://www.redhat.com/mailman/listinfo/pulp-list